scipy/sparse/linalg/isolve/tests/test_minres.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import division, print_function, absolute_import
+
+import numpy as np
+from numpy.testing import assert_almost_equal, run_module_suite
+
+from scipy.sparse.linalg import minres
+
+
+def test_minres():
+    a = np.random.randn(25).reshape(5,5)
+    b = np.random.randn(5)
+    c = np.random.randn(5)
+    x1 = minres(a, b)[0] + c
+    x2 = minres(a, b, x0=c)[0]
+    assert_almost_equal(x1, x2)
```

Comment:
This looks strange to me --- I'd expect the initial value to not affect the final result.
If you replace `minres` by eg. `gmres` the assert fails.

You'd expect `abs(a.dot(x) - b) < tol` for any `x0` provided.